portfolio/src/main/java/com/google/sps/servlets/RestaurantDataServlet.java

```diff
@@ -30,12 +30,11 @@
 /** Returns restaurant data as a JSON array, e.g. [{"lat": 38.4404675, "lng": -122.7144313}] */
 @WebServlet("/restaurant-data")
 public class RestaurantDataServlet extends HttpServlet {
-
-  private static final Map<Integer, String> NUMBERS = ImmutableMap.of(
-    1, "one", 2, "two", 3, "three");
+  private static final Map<Integer, String> NUMBERS =
+      ImmutableMap.of(1, "one", 2, "two", 3, "three");
   private Collection<Restaurant> restaurants;
   private String my_region = "California";
-  
+
   @Override
   public void init() {
     restaurants = new ArrayList<>();
```

Comment:
In separate pull request: you could use ImmutableList: https://guava.dev/releases/23.0/api/docs/com/google/common/collect/ImmutableList.html.